HW0/P6/P6B.py

```diff
@@ -33,4 +55,6 @@ def burnTime(t):
     plt.xlabel('Wait Time (sec)')
     plt.ylabel('Serial Time (sec) / Parallel Time (sec)')
     plt.title('Speedup versus function time')
+    plt.savefig('P6.png', dpi=200, bbox_inches='tight')
     plt.show()
+
```

Comment:
I had no useful tips to add for the code, it looks great. In general, I appreciated your detailed explanations in your text answers of how you came to your conclusions.